doc/references.bib

```diff
@@ -5,2003 +5,2003 @@
 
 % MNE-C reference
 @article{GramfortEtAl2014,
```

Comment:
@PABannier please revert the massive changes in this file.
you should not reformat files like this as it makes huge diffs